### PR TITLE
Add German translation for poll text

### PR DESCRIPTION
**Pull request by xMarcell**
New german lines for polls, GPed by asgdf

[ I was a bit confused by the trivia lines, since trivia games can only be started in the trivia room and the room does not have its language setting set to "german" , due to that it would not be possible for the server to translate them, so I left them open; if it's still possible and I am wrong, please let me know, then I will add the remaining lines, thanks :) ]

### CODE REVIEW

**Comment by AnnikaCodes** [comment]
> [ I was a bit confused by the trivia lines, since trivia games can only be started in the trivia room and the room does not have its language setting set to "german" , due to that it would not be possible for the server to translate them, so I left them open; if it's still possible and I am wrong, please let me know, then I will add the remaining lines, thanks :) ]

Trivia is translatable in case side servers that are in other languages want to play Trivia. It's not a big priority to translate it.

File: translations/german.json [hunk 1]
```diff
@@ -124,21 +124,21 @@
...
-		"Submit your vote": "",
-		"Quiz": "",
-		"Poll": "",
-		"Submit": "",
-		"ended": "",
-		"votes": "",
-		"delete": "",
+		"That option is not selected.": "Diese Wahloption ist nicht ausgew&auml;hlt.",
+		"You have already voted for this poll.": "Du hast bereits f&uuml;r diese Umfrage abgestimmt.",
+	  "No options selected.": "Keine Wahloption wurde ausgew&auml;hlt.",
```
Comment (Zarel) [inline translations/german.json:129]:
```suggestion
		"No options selected.": "Keine Wahloption wurde ausgew&auml;hlt.",
```

I believe you messed up the indentation for this one.

**Comment by Zarel** [comment]
Also, as mentioned in the other one, please don't use HTML entities. You can use ü or whatever in the translations directly.

**Comment by xMarcell** [comment]
Alright, I will avoid HTML entities in the future, thank you both

**Comment by xMarcell** [comment]
I commited your suggestion regarding the indentation, Zarel, is there anything else to do, or can this be merged now? Thank you

**Comment by Zarel** [comment]
You haven't changed the HTML entities yet.

**Comment by Zarel** [comment]
Specifically, you should be using `ä` instead of `&auml;`, and `ü` instead of `&uuml;`

**Comment by xMarcell** [comment]
Hey, there are other HTML entities from previous pull requests, shall I replace them aswell or leave them and just edit the ones from the present request? Sorry for asking too much questions, I am relatively new to this

**Comment by Zarel** [comment]
Yes, please do, thank you!

**Comment by xMarcell** [comment]
Removed any HTML entities from the file, should be good now

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**